versions.lock

```diff
@@ -369,8 +367,8 @@ org.apache.kerby:kerb-common:1.0.1 (2 constraints: a51841ca)
 org.apache.kerby:kerb-identity:1.0.1 (1 constraints: 5f0cb602)
 org.apache.kerby:kerb-server:1.0.1 (1 constraints: d10b65f2)
 org.apache.kerby:kerb-simplekdc:1.0.1 (1 constraints: dc0d7e3e)
+org.apache.tomcat.embed:tomcat-embed-el:9.0.63 (1 constraints: d01553cf)
```

Comment:
due to https://github.com/spring-projects/spring-boot/issues/24744

Reply:
replaces jakarta-el

Reply:
Where are we using this?

Reply:
its the same question about jakarata-el that is replaced. s3mock uses spring boot and only in tests.